pages/acountpage/admin.js

```diff
@@ -0,0 +1,52 @@
+import { useRouter } from "next/router";
+import React, { useState, useEffect } from "react";
+import Header from "../../components/layout/conponents/header/Header";
+import jwt from "jsonwebtoken";
+import TitleBlock from "../../scenes/Services/Components/TitleBlock";
+import Side from "../../scenes/Acount/Side";
```

Comment:
```suggestion
import { useRouter } from "next/router";
import React, { useState, useEffect } from "react";
import jwt from "jsonwebtoken";
import Header from "../../components/layout/conponents/header/Header";
import TitleBlock from "../../scenes/Services/Components/TitleBlock";
import Side from "../../scenes/Acount/Side";
```

сначала импорты из библиотеки, потом импорты компонентов